### ISSUE TITLE
Support iterations on groups

### ISSUE THREAD

**Issue by shans** [web-animations/web-animations-next#322]

**Comment by nordisk** [comment]
Why is this enhancement considered low priority?
One example: Group iterations would allow for creating slideshows or other looping animations with ```web-animations``` very easily.

**Comment by yepnamesjames** [comment]
Yeah, I +1 this. This enhancement would be extremely helpful for some basic product ideas.

**Comment by J7mbo** [comment]
This would be extremely useful for even basic groups of animations. Please consider this, most animations are not just basic, non-grouped ones.
